app/services/vendor_validator_service.rb

```diff
@@ -0,0 +1,45 @@
+class VendorValidatorService
```

Comment:
Instead of `VendorValidatorService#submit_job`, how about a class called `SubmitIdvJob` with a `call` method?